Resources/translations/messages.en.yml

```diff
@@ -1,33 +1,33 @@
 core:
     interface:
-        backToTop: back to top
-        loading: loading
-        collapse: collapse
+        backToTop: Back to top
```

Comment:
We gingen toch niet alle vertalingen omzetten naar hoofdletters? zie ook Framework channel in slack

Reply:
Ik dacht ook dat we afgesproken hadden:
* capitalize er uit
* translations in kleine letters
* ucfirst gebruiken voor labels (automatisch) en verder handmatig in de applicatie waar nodig
* als er speciale gevallen zijn zoals RefuInterim, die dan wel met hoofdletters in de translations steken.

Correct me if I'm wrong though

Reply:
Correct